### ISSUE TITLE
Any plan to support on the iPad?

### ISSUE THREAD

**Issue by clutterwhy** [artem-barmin/obsidian-block-drag-n-drop#15]
Just like the idea of this plugin and wonder if there is plans for mobile support. Thanks

**Comment by artem-barmin** [comment]
Yes, I have plans to support mobile. It should work out of box, just need to test it.

**Comment by ycbrd** [comment]
Does "mobile support" mean to support not only the iPad as in the title, but also Android devices at the same time?
(Sorry for I have no concept on this, but I do think I will need this plugin on the phone. And thanks for your work :-)

**Comment by joelostblom** [comment]
I just tried this on my Android phone and it is not working. I think this would be a great addition, allowing users to drag'n'drop list items on the phone similar to how google keep does it

**Comment by DaveScream** [comment]
Bump. Obsidian with google Keeps list dragging feature would be awesome!!

**Comment by Calorion** [comment]
+1
